app/models/reports/export_raw.rb

```diff
@@ -33,55 +33,55 @@ def default_report_hash
-        estimated_cost: -> (od) { as_currency(od.estimated_cost) },
-        estimated_subsidy: -> (od) { as_currency(od.estimated_subsidy) },
-        estimated_total: -> (od) { as_currency(od.estimated_total) },
-        actual_cost: -> (od) { as_currency(od.actual_cost) },
-        actual_subsidy: -> (od) { as_currency(od.actual_subsidy) },
-        actual_total: -> (od) { as_currency(od.actual_total) },
-        reservation_start_time: -> (od) { od.reservation.reserve_start_at if od.reservation },
-        reservation_end_time: -> (od) { od.reservation.reserve_end_at if od.reservation },
-        reservation_minutes: -> (od) { od.reservation.duration_mins if od.reservation },
-        actual_start_time: -> (od) { od.time_data.actual_start_at if od.time_data.present? },
-        actual_end_time: -> (od) { od.time_data.actual_end_at if od.time_data.present? },
-        actual_minutes: -> (od) { od.time_data.actual_duration_mins if od.time_data.present? },
-        canceled_at: -> (od) { od.reservation.canceled_at if od.reservation },
-        canceled_by: -> (od) { canceled_by_name(od.reservation) if od.reservation },
+        bundled_products: ->(od) { od.product.is_a?(Bundle) ? od.product.products.collect(&:name).join(" & ") : nil },
+        account_type: ->(od) { od.account.type.underscore.humanize },
+        affiliate: ->(od) { od.account.affiliate_to_s },
+        account: ->(od) { od.account.account_number },
+        account_description: ->(od) { od.account.description_to_s },
+        account_expiration: ->(od) { od.account.expires_at },
+        account_owner: ->(od) { od.account.owner_user.username },
+        owner_first_name: ->(od) { od.account.owner_user.first_name },
+        owner_last_name: ->(od) { od.account.owner_user.last_name },
+        owner_email: ->(od) { od.account.owner_user.email },
+        price_group: ->(od) { od.price_policy.try(:price_group).try(:name) },
+        estimated_cost: ->(od) { as_currency(od.estimated_cost) },
+        estimated_subsidy: ->(od) { as_currency(od.estimated_subsidy) },
+        estimated_total: ->(od) { as_currency(od.estimated_total) },
+        actual_cost: ->(od) { as_currency(od.actual_cost) },
+        actual_subsidy: ->(od) { as_currency(od.actual_subsidy) },
+        actual_total: ->(od) { as_currency(od.actual_total) },
+        reservation_start_time: ->(od) { od.reservation.reserve_start_at if od.reservation },
+        reservation_end_time: ->(od) { od.reservation.reserve_end_at if od.reservation },
+        reservation_minutes: ->(od) { od.reservation.duration_mins if od.reservation },
+        actual_start_time: ->(od) { od.time_data.actual_start_at if od.time_data.present? },
+        actual_end_time: ->(od) { od.time_data.actual_end_at if od.time_data.present? },
+        actual_minutes: ->(od) { od.time_data.actual_duration_mins if od.time_data.present? },
+        canceled_at: ->(od) { od.canceled_at },
```

Comment:
This is totally fine, but fyi, if it's a method call directly on the order detail, you can just use the symbol `canceled_at: :canceled_at` rather than a proc.